cccm-backend/cccm-api/src/main/java/ca/bc/gov/open/jag/api/service/ClientFormSaveServiceImpl.java

```diff
@@ -48,16 +47,8 @@ public BigDecimal createSARA(CreateFormInput createFormInput, BigDecimal locatio
     }
 
     @Override
-    public BigDecimal updateForm(UpdateFormInput updateFormInput, BigDecimal locationId) {
-
-        FormInput formInput = new FormInput();
-        formInput.setLocationId(locationId);
-        formInput.setClientFormId(updateFormInput.getClientFormId());
-        formInput.setPlanSummary(updateFormInput.getPlanSummary());
-        formInput.setFormLevelComments(updateFormInput.getReAssessmentComments());
-        formInput.setSourcesContacted(updateFormInput.getSourcesContacted());
-
-        return obridgeClientService.createForm(formInput);
+    public void updateForm(BigDecimal clientFormId, String updateFormInput) {
```

Comment:
This was for updating the top level form information. Will those fields not be updated?

Reply:
Or should this actually be two calls?

Reply:
Moved the updateForm endpoint to tests group as discussed.